src/vma/iomux/epfd_info.cpp

```diff
@@ -739,12 +740,18 @@ void epfd_info::statistics_print(vlog_levels_t log_level /* = VLOG_DEBUG */)
 	vlog_printf(log_level, "Fd number : %d\n", m_epfd);
 	vlog_printf(log_level, "Size : %d\n", m_size);
 
-	for (int i = 0 ; i < m_n_offloaded_fds ; i++) {
-		offloaded_str_place += snprintf(&offloaded_str[offloaded_str_place] , offloaded_str_cell_size, " %d ", m_p_offloaded_fds[i]);
-		offloaded_str_place--;
+	vlog_printf(log_level, "Offloaded Fds : %d\n", m_n_offloaded_fds);
+
+	while (i < m_n_offloaded_fds) {
+		memset(offloaded_str, 0, sizeof(offloaded_str));
+		for (offloaded_str_place = 0; offloaded_str_place < EPFD_MAX_OFFLOADED_STR && i < m_n_offloaded_fds; i++) {
+			offloaded_str_place += snprintf(&offloaded_str[offloaded_str_place], sizeof(offloaded_str) - offloaded_str_place - 1, " %d", m_p_offloaded_fds[i]);
```

Comment:
snprintf() can return negative value in case error. So please consider more stable solution.
Example:
```
	int n = 0;
	n = snprintf();
	if (!likely((0 < n) && (n < (int)(sizeof(offloaded_str) - offloaded_str_place - 1)))) {
		break;
	}
```